src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid implements Figure {
+    private int top;
+    private int bottom;
+    private int height;
+    private String color;
+
+    public IsoscelesTrapezoid() {
+        this.top = top;
+        this.bottom = bottom;
+        this.height = height;
+        this.color = color;
+    }
+
+    @Override
+    public double getArea() {
+        return (top + bottom) / 2 * height;
```

Comment:
The formula for the area calculation is incorrect because the division by 2 has higher precedence than addition, which will lead to an incorrect result. You need to add parentheses around (top + bottom) to correctly calculate the average of the two bases before multiplying by the height.